Sources/Kevin/Accounts/KevinAccountsPlugin.swift

```diff
@@ -32,6 +32,13 @@ public class KevinAccountsPlugin: KevinPlugin {
         return configuration.callbackUrl
     }
     
+    public func isShowUnsupportedBanks() throws -> Bool {
```

Comment:
`shouldExcludeBanksWithoutAccountLinkingSupport()` and also change condition then